TSC-Charter.md

```diff
@@ -56,14 +57,32 @@ one-fourth of the TSC membership shares an employer, then the situation
 must be immediately remedied by the resignation or removal of one or more
 TSC members affiliated with the over-represented employer(s).
 
-The TSC members shall consist of Maintainers from Core Projects as
-defined in the project lifecycle document and Section 7.
+The TSC may, at its discretion, invite any number of non-voting observers to
+participate in the public portion of TSC discussions and meetings.
 
 The TSC shall meet regularly using tools that enable participation by the
 community (e.g. weekly on a Google Hangout On Air, or through any other
 appropriate means selected by the TSC). The meeting shall be directed by
-the TSC Chairperson. Minutes or an appropriate recording shall be taken
-and made available to the community through accessible public postings.
+the TSC Chairperson. Responsibility for directing individual meetings may be
+delegated by the TSC Chairperson to any other TSC member. Minutes or an
+appropriate recording shall be taken and made available to the community
+through accessible public postings.
+
+TSC members are expected to regularly participate in all TSC activities
+including:
+
+* Regular attendance in TSC meetings;
+* Actively participating in TSC related discussions on GitHub; and
+* Voting
+
+In the case where an individual TSC member: (a) attends fewer than 25% of the
+regularly scheduled meetings held within a six month period, (b) does not attend
+at least one meeting every three months, (c) does not regularly participate in
```

Comment:
As well, this section could be clearer if the TSC member needs to meet _any_ or _all_ of the requirements.  There is an "and" buried in there but it's rather subtle and it could be interpreted as combining only (c) and (d).